docker/development/debian/Dockerfile

```diff
@@ -49,11 +49,18 @@ RUN git clone --branch 3.28 --depth 1 https://github.com/lucas-bremond/geometric
 
 ## Open Space Toolkit ▸ Core [0.3.0]
 
-RUN mkdir -p /tmp/open-space-toolkit-core \
- && cd /tmp/open-space-toolkit-core \
- && wget --quiet https://github.com/open-space-collective/open-space-toolkit-core/releases/download/0.3.0/open-space-toolkit-core-0.3.0-1.x86_64-runtime.deb \
- && wget --quiet https://github.com/open-space-collective/open-space-toolkit-core/releases/download/0.3.0/open-space-toolkit-core-0.3.0-1.x86_64-devel.deb \
- && apt-get install -y ./*.deb \
+# RUN mkdir -p /tmp/open-space-toolkit-core \
+#  && cd /tmp/open-space-toolkit-core \
+#  && wget --quiet https://github.com/open-space-collective/open-space-toolkit-core/releases/download/0.3.0/open-space-toolkit-core-0.3.0-1.x86_64-runtime.deb \
+#  && wget --quiet https://github.com/open-space-collective/open-space-toolkit-core/releases/download/0.3.0/open-space-toolkit-core-0.3.0-1.x86_64-devel.deb \
+#  && apt-get install -y ./*.deb \
+#  && rm -rf /tmp/open-space-toolkit-core
+
+# TBR: for assessing cross-use of ostk packages with the use of pybind11
```

Comment:
TBR, same as for binaries. Kept to allow for tests with consistent dependencies' versions but should not be here